<launch>

	<arg name="joy_topic" default="/arm/joy"/>

	<!-- The hockey_stick_joint node will take arm joint velocity commands and convert them to motor commands -->
	<node name="hockey_stick_controller" pkg="hockey_stick_arm" type="hockey_stick_joint.py">
		<param name="arm_joy_cmd" value="$(arg joy_topic)"/>
	</node>

	<!-- We need to run myactuator_control to run the motors -->
	<include file="$(find hockey_stick_arm)/launch/myactuator_arm.launch">
		<arg name="output" value="screen"/>
	</include>

</launch>
